sormas-backend/src/main/java/de/symeda/sormas/backend/campaign/CampaignFacadeEjb.java

```diff
@@ -193,22 +192,68 @@ public Campaign fromDto(@NotNull CampaignDto source) {
 					.map(campaignFormMetaReferenceDto -> campaignFormMetaService.getByUuid(campaignFormMetaReferenceDto.getUuid()))
 					.collect(Collectors.toSet()));
 		}
-		target.setDashboardElements(source.getCampaignDashboardElements());
-
+		final List<CampaignDashboardElement> campaignDashboardElements = source.getCampaignDashboardElements();
+		if (campaignDashboardElements != null) {
+			target.setDashboardElements(
+				campaignDashboardElements.stream()
+					.map(
+						cdewc -> new CampaignDashboardElement(
+							cdewc.getDiagramId(),
+							cdewc.getTabId(),
+							cdewc.getOrder(),
+							cdewc.getWidth(),
+							cdewc.getHeight()))
+					.collect(Collectors.toList()));
+		}
 		return target;
 	}
 
+	public void validate(CampaignReferenceDto campaignReferenceDto) {
+		this.validate(getByUuid(campaignReferenceDto.getUuid()));
```

Comment:
"this." is unnecessary and might actually be wrong because we're in a stateless facade